tests/test_openjdk.py

```diff
@@ -28,3 +56,38 @@ def test_jdk_version(container, java_version):
     assert (
         container.connection.check_output("echo $JAVA_VERSION") == java_version
     )
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    CONTAINER_IMAGES_EXTENDED,
+    indirect=["container_per_test"],
+)
+@pytest.mark.parametrize(
+    "test_to_run, expected_strings",
+    [
+        (
+            "threads_concurrency_and_sleep",
+            ["I am the thread 1", "I am the thread 2"],
+        )
+    ],
+)
+def test_jdk_extended(
+    container_per_test,
+    test_to_run: str,
+    expected_strings: list[str],
+    host,
+    container_runtime: OciRuntimeBase,
+):
+    """Executes a set of java tests stored on test/trainers/java.
+    The check is a list of expected strings.
```

Comment:
This does not clarify what the test executes

Reply:
better now?